### PR TITLE
[Method References 2] LPS-169777 Remove class ScriptingExecutorExtender

### PR DESCRIPTION
**Pull request by W2jx**

### CODE REVIEW

**Comment by Kvale1733** [comment]
@W2jx No regressions found. But the SF issue is real. ScriptingExecutorExtender.java needs to be removed from source-formatter.properties.

**Comment by W2jx** [comment]
ci:test:sf

**Comment by W2jx** [comment]
no conflicts(6/2 8:25)

**Comment by tinatian** [comment]
@W2jx Please add breaking change.

**Comment by tinatian** [comment]
@W2jx Is it safe to remove this, I found a bug ticket might be related to it https://issues.liferay.com/browse/LPS-66457.

If we really want to remove the feature, then this should be added to breaking change.

FYI @dantewang 

Tina.

**Comment by W2jx** [comment]
Hi @tinatian , We have confirmed with the TS team. I have added the breaking change markdown.

**Comment by W2jx** [comment]
no logic changes, no need to rerun test

File: readme/BREAKING_CHANGES.markdown [hunk 1]
```diff
@@ -1379,4 +1379,27 @@ Replace `getParameter71()` with `getParameterOptional()`, `getParameterValues71(
...
+
+`ScriptingExecutorExtender` class and `ScriptBundleProvider` interface were removed.
+
+### Who is affected?
+
+This affects anyone implementing this interface: `ScriptBundleProvider`, and having some script files in this path: `/META-INF/resources/scripts/` in the same module.
+
+### How should I update my code?
+
+Delete implementations of `ScriptBundleProvider` and remove script files in `/META-INF/resources/scripts/`.
```
Comment (tinatian) [inline readme/BREAKING_CHANGES.markdown:1401]: > Delete implementations of `ScriptBundleProvider` and remove script files in `/META-INF/resources/scripts/`.
What about the logic I have in the script files? Simply deleting will affect behaviors.

File: readme/BREAKING_CHANGES.markdown [hunk 2]
```diff
@@ -1379,4 +1379,27 @@ Replace `getParameter71()` with `getParameterOptional()`, `getParameterValues71(
...
+
+### Why was this change made?
+
+`ScriptBundleProvider` is `ProviderType` but no one is implementing it in Liferay.
```
Comment (tinatian) [inline readme/BREAKING_CHANGES.markdown:1405]: @W2jx @dantewang This is not a good reason to remove. What did TS told you?

**Pull request closed without merging**